### PR TITLE
Handle dark themed assets

### PR DESCRIPTION
**Pull request by oa495**
- New config props for Flavor and Asset: `colorScheme` and `imageset`
- `imageset` is used to group light and dark versions of an asset under the defining name.
- Set `imageset` prop on raster (webp|png) flavors 
- [ios] Distribute dark and light assets into shared `imageset` folder (don't treat dark assets as a separate variant)
- Replacing some Linkedin assets with fake ones

### CODE REVIEW

File: packages/@icon-magic/generate/test/fixtures/out/home-2/ads/build/iconrc.json [hunk 1]
```diff
@@ -37,12 +37,14 @@
       "name": "active-small",
       "path": "./active-small.svg",
       "imageset": "active-small",
+      "colorScheme": "light",
```
Comment (ramitha) [inline packages/@icon-magic/generate/test/fixtures/out/home-2/ads/build/iconrc.json:31]: Do we have to have this? Can it be assumed to default to `light`, if there's a corresponding asset with the same imageset that's `dark`?

File: packages/@icon-magic/generate/src/plugins/svg-to-raster.ts [hunk 2]
```diff
@@ -186,12 +185,12 @@ export const svgToRaster: GeneratePlugin = {
...
             path: `./${assetName}.png`,
-            imageset: imageset
+            imageset: imageset,
+            colorScheme: flavor.colorScheme
           },
           webp: {
             name: assetName,
-            path: `./${assetName}.webp`,
-            imageset: imageset
+            path: `./${assetName}.webp`
```
Comment (ramitha) [inline packages/@icon-magic/generate/src/plugins/svg-to-raster.ts]: Good to propagate the colorScheme and imageset values to webp as well, even though it isn't used anywhere as of now

**Review by ramitha — APPROVED** [review]
Small suggestion, otherwise it looks good. Also, you can disregard my comment above, `light` `colorScheme` doesn't look mandatory.

File: packages/@icon-magic/generate/test/fixtures/out/home-2/ads/build/iconrc.json (hunk 1 above)
Comment (oa495) [reply packages/@icon-magic/generate/test/fixtures/out/home-2/ads/build/iconrc.json:31]: we don't, we can assume as you mentioned. it's not actually used for anything at this point, I just did here as an example.

**Pull request merged**